System.Drawing: Consolidate Clip-related code in Graphics

qmfrederik opened the pull request:
- Move common Clip-related code from Graphics.Unix and Graphics.Windows to Graphics
- Move P/Invoke methods to GdiplusNative

mellinoe commented on src/System.Drawing.Common/src/System/Drawing/Graphics.Unix.cs:
```diff
@@ -2452,14 +2218,14 @@ public RectangleF VisibleClipBounds
             }
         }
-
+ 
```
nit: remove extra whitespace

mellinoe commented on src/System.Drawing.Common/src/System/Drawing/Graphics.Windows.cs:
```diff
@@ -3434,126 +3315,23 @@ public object GetContextInfo()
                 }
-
-                var rect = new GPRECTF();
+		
```
nit: Extra whitespace here and below.

mellinoe replied:
@qmfrederik It looks like there is still some extra whitespace in this method. Could you clean it up?

qmfrederik replied:
@mellinoe Oops.. hope I got it right this time.

mellinoe commented on src/System.Drawing.Common/src/Unix/System.Drawing/Region.cs:
```diff
@@ -642,6 +642,19 @@ internal IntPtr NativeObject
             }
         }
+
+        internal IntPtr _nativeRegion
```
Are we not able to just rename the field to `_nativeRegion`?

qmfrederik replied:
#23189 removed the Unix implementation of `Region`, so this problem fixed itself ;-)

mellinoe commented:
(Also needs a rebase)

qmfrederik commented:
@mellinoe I rebased and updated the PR according to the feedback; should be good now. I'll keep an eye on the CI to see if something broke as part of the rebase, though.

qmfrederik commented:
@dotnet-bot test Linux x64 Release Build
